docs/source/ExoCTK.atmospheric_retrievals.rst

```diff
@@ -0,0 +1,22 @@
+ExoCTK.atmospheric\_retrievals package
+======================================
+
+Submodules
+----------
+
+ExoCTK.atmospheric\_retrievals.platon_wrapper module
```

Comment:
@julesfowler Good catch.  I think things got messed up after I updated this branch with changes from #235

Reply:
@julesfowler And as far as documenting hidden functions -- good questions.  I'm not sure how we want to handle this situation as a group.  I don't think there is an easy way in `sphinx` autodoc to tell it to ignore functions that begin with underscores.

Reply:
@julesfowler Actually upon further investigation, it appears that `sphinx` will automatically hide private functions, unless otherwise specified.  So I don't think we have to change anything here.